app/dashboard/migrations/0054_auto_20190930_1525.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 2.2.4 on 2019-09-30 15:25
+
+from django.db import migrations, models
```

Comment:
seems to me there is a problem with this migration, since already exist on master but for activities